front/components/assistant/conversation/SidebarMenu.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button, ChatBubbleBottomCenterPlusIcon, Item } from "@dust-tt/sparkle";
+import { useRouter } from "next/router";
+
+import { useConversations } from "@app/lib/swr";
+import { ConversationWithoutContentType } from "@app/types/assistant/conversation";
+import { WorkspaceType } from "@app/types/user";
+
+export function AssistantSidebarMenu({ owner }: { owner: WorkspaceType }) {
+  const router = useRouter();
+
+  const onNewConversation = async () => {
+    void router.push(`/w/${owner.sId}/assistant/new`);
+  };
+
+  // @todo move to a lib function?
+  const isWorkspaceUpgraded =
+    owner.plan !== null &&
+    typeof owner.plan?.limits?.dataSources?.count === "number" &&
+    owner.plan?.limits?.dataSources?.count !== 0;
+
+  const { conversations } = useConversations({ workspaceId: owner.sId });
+
+  return (
+    <div className="flex grow flex-col">
+      <div className="flex h-0 min-h-full w-full overflow-y-auto">
+        <div className="flex w-full flex-col pl-4 pr-2">
+          <div className="pr py-4 text-right">
+            <Button
+              disabled={!isWorkspaceUpgraded}
```

Comment:
You can remove this. non upgraded worskapces will have access to @gpt3.5-turbo and @claude-instant